pkg/policy/api/fqdn.go

```diff
@@ -51,7 +51,7 @@ type FQDNSelector struct {
 	// A trailing "." is automatically added when missing.
 	//
 	// Examples:
-	// `*.cilium.io` matches subomains of cilium at that level
+	// `*.cilium.io` matches subdomains of cilium at that level
```

Comment:
You'll need to run `make generate-k8s-api && make manifests` and amend the result to the commit to update the generated files as well and fix the [failing linter workflow](https://github.com/cilium/cilium/actions/runs/12838797988/job/35805049249?pr=37060).